gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.7-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.10-bin.zip
```

Comment:
Sorry, hadn't seen this PR.

The version also needs to be bumped in 3 places in `.github/workflows/build.yml`.

Reply:
I did not know this needed to be updated. Can you please provide more information on what needs to be changed in this file?

Reply:
The version, just like in #1417.

Reply:
These Gradle version lines were removed in #1421. Do you think they should be readded?

Reply:
Oh, had missed that, that's nice! Do we get the current in-tree version of gradle automatically now?

Reply:
I think so, but if not, it will be downloaded automatically, so I am not sure.

Reply:
Well, merge it then, worst case is we have to update something more later!